pydrad/parse/parse.py

```diff
@@ -172,10 +171,15 @@ def initial_conditions(self):
         `~pydrad.parse.Profile` for the solutions to the hydrostatic
         equations used as initial conditions.
         """
+        profile_kwargs = self._profile_kwargs.copy()
+        # NOTE: These files do not exist for the initial conditions
+        profile_kwargs['read_hstate'] = False
+        profile_kwargs['read_ine'] = False
+        profile_kwargs['read_trm'] = False
```

Comment:
If the files exist, I would think it would be better to default to True.  Perhaps a quick check for `profile0.hstate`, etc., and then set the default from there?  Or would that be too much overhead?

Reply:
In the `Profile` constructor, these are all set to True by default so that the files are read if they exist. These are only set to False for the initial conditions because, to my knowledge, there are no versions of these files for the initial conditions.

Reply:
Oh, yes, these files are not printed out for the initial conditions.  I missed that this was in reference to the initial conditions.